CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+Contributing to the Majora's Mask Decompilation Project
+=======================================================
+
+Thanks for helping us reverse engineer *The Legend of Zelda: Majora's Mask* for the N64! 
+All contributions are welcome. This is a group effort, and even small contributions can make a difference. Some tasks also don't require much knowledge to get started.
+
+This document is meant to be a set of tips and guidelines for contributing to the project.
+For general information about the project, see [our readme](https://github.com/zeldaret/mm/blob/master/README.md) or [our wiki](https://github.com/zeldaret/mm/wiki).
+
+Most discussions happen on our [Discord Server](https://discord.zelda64.dev), where you are welcome to ask if you need help getting started, or if you have any questions regarding this project and other decompilation projects.
+
+
+Useful Links
+------------
+
+- [Zelda 64 Reverse Engineering Website](https://zelda64.dev/) - Our homepage, with links to other resources.
+- [Installation guide](https://github.com/zeldaret/mm/wiki/Installation) - Instructions for getting this repository set up and built on your machine.
+- [Introduction to OOT decomp](https://github.com/zeldaret/oot/blob/master/docs/tutorial/contents.md) - A very detailed tutorial on how to get started with decomp. For OOT, but largely applicable to MM as well.
+
+- [Discord](https://discord.zelda64.dev/) - Primary discussion platform.
+- [Trello board](https://trello.com/b/ruxw9n6m/majoras-mask-decompilation) - We use this to track decompilation progress, not GitHub Issues.
+
+Getting Started
+---------------
+
+### What should I know to take part?
+
+Basic knowledge of C, particularly arrays and pointers, is extremely useful. Knowledge of MIPS is not required initially, you can pick it up as you work through some decompilation.
+
+You should be familiar with using git and GitHub. There are a number of tutorials available online, [such as this one](https://github.com/firstcontributions/first-contributions), which can help you get started.
+
+The most useful knowledge to have is a general understanding of how the game works. An afternoon of constructive mucking about in the [OOT Practice Rom](http://practicerom.com/) (aka GZ) or the [MM Practice Rom](https://kz.zeldacodes.org/) (aka KZ) will be very beneficial if you have not looked at either of the N64 Zelda's internals before.
```

Comment:
I would call people to look at OOT's engine code here since that decompilation is much further along, and is a great reference in general for working in MM.